$NetBSD$

The -ansi compiler option implies that JSON_ISO_STRICT is turned on.

--- avidemux_core/ADM_coreUtils/src/CMakeLists.txt.orig	2022-09-17 13:59:17.000000000 +0000
+++ avidemux_core/ADM_coreUtils/src/CMakeLists.txt
@@ -28,7 +28,8 @@ ADM_last.cpp
 )
 
 include_directories(../include Source "${PTHREAD_INCLUDE_DIR}")
-add_definitions("-std=c++11")
+add_compile_options(-std=c++11)
+add_compile_definitions(JSON_ISO_STRICT)
 ADM_ADD_SHARED_LIBRARY(ADM_coreUtils6 ${ADM_coreUtils_SRCS})
 TARGET_LINK_LIBRARIES(ADM_coreUtils6 ADM_json ADM_core6 ADM_coreUI6 ADM_libavutil ADM_libavcodec )
 
